Makefile

```diff
@@ -5,7 +5,7 @@
 ERLC ?= erlc
 DIALYZER ?= dialyzer
 
-EFLAGS += -DVSN='"$(VSN)"' +debug_info +warn_exported_vars +warn_unused_vars +warn_unused_import +warn_missing_spec
+EFLAGS += -DVSN='"$(VSN)"' +debug_info +warn_exported_vars +warn_unused_vars +warn_unused_import +nowarn_missing_spec
```

Comment:
In Erlang/OTP 19.3, the oldest version I ran this on, there were already complaints of missing specs. My goal was not to fix everything, but to, at least, spruce it up a little.